modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -762,12 +762,41 @@ public ObjectEntry partialUpdateObjectEntry(
 		}
 
 		if (objectEntry.getProperties() != null) {
-			Map<String, Object> properties =
+			Map<String, Object> existingProperties =
 				existingObjectEntry.getProperties();
 
-			properties.putAll(objectEntry.getProperties());
+			Map<String, Object> properties = objectEntry.getProperties();
 
-			existingObjectEntry.setProperties(() -> properties);
+			for (ObjectRelationship objectRelationship :
+					_objectRelationshipLocalService.
+						getObjectRelationshipsByObjectDefinitionId2(
+							objectDefinition.getObjectDefinitionId())) {
+
+				ObjectField objectField =
+					objectFieldLocalService.getObjectField(
+						objectRelationship.getObjectFieldId2());
+
+				String objectRelationshipERCObjectFieldName =
+					ObjectFieldSettingUtil.getValue(
+						ObjectFieldSettingConstants.
+							NAME_OBJECT_RELATIONSHIP_ERC_OBJECT_FIELD_NAME,
+						objectField);
+
+				if (properties.containsKey(objectField.getName()) &&
+					!properties.containsKey(
+						objectRelationshipERCObjectFieldName)) {
+
+					existingProperties.remove(
+						ObjectFieldSettingUtil.getValue(
+							ObjectFieldSettingConstants.
+								NAME_OBJECT_RELATIONSHIP_ERC_OBJECT_FIELD_NAME,
+							objectField));
```

Comment:
@p-albuquerque you should reuse the existing `objectRelationshipERCObjectFieldName` variable here